File: 2058-concatenation-of-array/2058-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        count = 0
        ans = []
        while count < 2:
            for i in nums:
                ans.append(i)
            count += 1;
        return ans

File: 2058-concatenation-of-array/2058-concatenation-of-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {


    // let numberOfIncrementOverTheWholeNumsArray =0;
    // // let index = 0;
    // let ans = [...nums, ...nums];


    // // while(numberOfIncrementOverTheWholeNumsArray < 2 ){

    // //     ans.push(nums[index])

    // //     index+=1;
    // //     if(index === nums.length){
    // //         index = 0;
    // //         numberOfIncrementOverTheWholeNumsArray+= 1;
    // //     }
    // // }
    // return ans;

    let ans = [];

    for(let i = 0; i < 2; i++){
        for(let i = 0; i < nums.length; i++){
            ans.push(nums[i]);
        }
    }

    return ans;
};